spec/12-the-scala-standard-library.md

```diff
@@ -642,6 +642,12 @@ object Predef {
   def classOf[T]: Class[T] = null
    // this is a dummy, classOf is handled by compiler.
 
+  // valueOf -----------------------------------------------------------
+
+  /** Retrieve the single value of a type with a unique inhabitant. */
+  @inline def valueOf[T](implicit vt: ValueOf[T]): T {} = vt.value
```

Comment:
`T {}` is a little confusion because it does not respect `.widen`:

``` scala
@ val t = typeOf[0 {}] 
t: Type = RefinedType(List(FoldableConstantType(Constant(0))), SynchronizedOps())
@ val w = t.widen 
w: Type = RefinedType(List(FoldableConstantType(Constant(0))), SynchronizedOps())
@ println(t eq w) 
true
```